Goobi/src/de/sub/goobi/metadaten/Paginator.java

```diff
@@ -99,9 +99,20 @@ private void assertSelectionIsNotNull() {
 		}
 	}
 
+	/**
+	 * Throws NumberFormatException if `paginationStartValue` isn’t a valid
+	 * number of the type specified by `paginationType`.
+	 * 
+	 * @throws NumberFormatException
+	 *             if `paginationStartValue` isn’t valid
+	 */
 	private void assertValidPaginationStartValue() {
 		// arabic numbers
 		if (paginationType == Paginator.Type.ARABIC) {
+			/* coverity[USELESS_CALL]
+			 * Integer.parseInt() throws NumberFormatException if
+			 * paginationStartValue cannot be parsed to an `int`, and this is
+			 * what we want here. */
```

Comment:
No idea, I cannot test it anyway. However, Joshua Bloch [writes](https://jtechies.blogspot.de/2012/07/item-62-document-all-exceptions-thrown.html) in Effective Java: “Use the Javadoc @throws tag to document each unchecked exception that a method can throw, but do _not_ use the throws keyword to include unchecked exceptions in the method declaration.” I followed his recommendation.